src/main/java/sprouts/Val.java

```diff
@@ -717,11 +716,12 @@ default boolean is( Val<T> other ) {
 	 * @return The truth value determining if the provided item is equal to the wrapped item.
 	 */
 	@SuppressWarnings("unchecked")
-	default boolean isOneOf( @Nullable T first, @Nullable T second, T... otherValues ) {
-		if ( this.is(first) ) return true;
-		if ( this.is(second) ) return true;
-		for ( T otherValue : otherValues )
-			if ( this.is(otherValue) ) return true;
+	default boolean isOneOf( @Nullable T first, @Nullable T second, @Nullable T @NonNull... otherValues ) {
```

Comment:
I just noticed, although the items inside the args array may be null, 
the array itself should not be null. So I would also add an `Objects.requireNonNull` for that here...